kahuna/public/js/components/gr-tooltip/gr-tooltip.js

```diff
@@ -20,6 +18,21 @@ tooltip.directive('grTooltip', ['onValChange', function (onValChange) {
                     element.attr('data-title', newTooltip);
                 }));
             }
+
+            element.bind('mouseenter', () => {
+                $scope.addTimer = $timeout(() => {
```

Comment:
Why delay the display of the tooltip? I thought one of the main advantages of this library was to be able to show them without delay, unike `title` attributes...?